scripts/check_enforcer/check_enforcer.py

```diff
@@ -279,7 +279,7 @@ def run_checks():
     if failed_reason != "":
         raise Exception(failed_reason)
 
-    # Loop for 15 minutes at most.
+    # Loop for 30 minutes at most.
```

Comment:
Why would our test take so long? given we have recording?
Maybe make it 20 mins. And make sure we won't exceed this limit.

Reply:
Maybe split sdk tests to more pipelines is a way to do this.

Reply:
checked two workflow runs:

- https://github.com/microsoft/promptflow/actions/runs/8643285277/job/23696078691: 515 tests, 11min
- https://github.com/microsoft/promptflow/actions/runs/8737000227/job/23973020515: 658 tests, 16min